client/src/components/Admin/OrganizationEdit.js

```diff
@@ -53,10 +53,19 @@ const validationSchema = Yup.object().shape({
   hours: Yup.array().of(HourSchema),
   twitter: Yup.string()
     .matches(
-      /^https?:\/\/(www\.)?(twitter\.com|x\.com)\/.*/,
-      "Invalid URL, e.g. 'https://twitter.com/ or https://x.com/'"
+      /^@([a-zA-Z][a-zA-Z0-9_]{0,14}|[0-9][a-zA-Z0-9_]{0,14})$/,
+      "Valid Twitter username required."
     )
-    .required("Full Twitter/X URL is required."),
+    .required("Valid Twitter username is required."),
```

Comment:
I don't think these fields should be required 🤔 they should be optional 👌

Reply:
okay, i fixed the validation to make it optional and also changed the design for both the pinterest and twitter.
<img width="1275" alt="Screenshot 2024-08-21 at 4 24 54 PM" src="https://github.com/user-attachments/assets/c32e2948-fabf-478d-bc61-975bc9fa8216">